pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2ClientConfiguration.java

```diff
@@ -19,6 +22,9 @@
  * @since 1.7
  */
 public final class SAML2ClientConfiguration implements Cloneable {
+	private KeyStore keyStore;
+
+	private Resource keystoreResource;
 
     private String keystorePath;
```

Comment:
Sure. But, I imagine you can remove the internal `keystorePath ` property and keep the setter (and getter) to initialize a specific file resource in the `keystoreResource` based on the provided path: it would be better: backward compatible and cleaner a the same time.